harness/determined/cli/tensorboard.py

```diff
@@ -206,6 +215,14 @@ def open_tensorboard(args: Namespace) -> None:
                     ),
                 ],
             ),
+            Cmd(
+                "delete",
+                delete_tensorboard,
+                "delete TensorBoard files associate with the proived experiment ID",
```

Comment:
@azhou-determined or someone from ml sys should review this change.

personally i think it could be a bit confusing for a user. maybe `det e delete-tensorboard-files :ID` is more clear about what it does but a pain to type.

"tensorboard" in this context so far means "a running tensorboard application" not "tensorboard files" so "det tensorboard delete ..." makes me think it would do something similar to "det tensorboard kill ...".

Reply:
agreed that `det tensorboard delete` reads like you want to delete the tensorboard instance.

maybe something like `delete-files`, or `delete-experiment`?

Reply:
i like `det tensorboard delete-files --experiment-id $ID` or `det experiment delete-tensorboard-files $ID`. preferring the latter.

Reply:
From offline discussion with @wes-turner & @stoksc. We went with `det e delete-tb-files`